### PR TITLE
[SDK-744] - Remove webview from the core

### PR DESCRIPTION
**Pull request by rk132**
## Description

-   Remove the Webview package installation from the core

## How to Test

-   Import the core package and observe, that the Webview is not installed anymore.

[SDK-744]: https://ready-player-me.atlassian.net/browse/SDK-744?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: Editor/Core/Scripts/ModuleManagement/ModuleList.cs [hunk 1]
```diff
@@ -53,14 +39,6 @@ public static class ModuleList
             var installedModules = new Dictionary<string, string>();
             installedModules.Add(Core.name, Core.version);
 
-            foreach (ModuleInfo module in Modules)
```
Comment (HarrisonHough) [inline Editor/Core/Scripts/ModuleManagement/ModuleList.cs:56]: I think removing this (and WebView) from module list will cause us to lose tracking of whether WebView has been installed.

**Review by HarrisonHough — CHANGES_REQUESTED** [review]
Overrall it looks fine however you might need to add back the WebView to module list so that we can track whether or not it is installed? 
Unless we try do it some other way from inside WebView package itself

File: Editor/Core/Scripts/ModuleManagement/ModuleList.cs (hunk 1 above)
Comment (rk132) [reply Editor/Core/Scripts/ModuleManagement/ModuleList.cs:56]: Yeah, makes sense to have it as it is already in place

**Review by HarrisonHough — APPROVED** [review]
Looks good now, nice work.

**Pull request merged**